python/ray/dashboard/client/src/pages/dashboard/Dashboard.tsx

```diff
@@ -87,7 +87,7 @@ class Dashboard extends React.Component<
     ];
 
     // if Tune information is not available, remove Tune tab from the dashboard
-    if (!tuneAvailability) {
+    if (tuneAvailability === null || !tuneAvailability["available"]) {
```

Comment:
Prefer the use of `tuneAvailability.available` rather than `tuneAvailability["available"]` whenever possible (i.e. when the property's name is a valid JavaScript identifier). Search `["` on the diff for this PR to find the other instances of this.